packages/maskbook/src/components/InjectedComponents/PayloadReplacer.tsx

```diff
@@ -12,16 +12,25 @@ const useStyle = makeStyles((theme: Theme) => ({
 
 export interface PayloadReplacerProps {
     payload: string
+    theme: Theme
 }
 
-export function PayloadReplacer({ payload }: PayloadReplacerProps) {
+export const PayloadReplacer = withTheme(function ({ payload, theme }: PayloadReplacerProps) {
```

Comment:
Why not `useTheme` hook? It is same as `withTheme`

Reply:
done